module/js/Main.js

```diff
@@ -8,15 +8,24 @@ class Util {
 	];
```

Comment:
this giant `Main` file is way past the point of being out of hand; next job is to split it up into multiple files + bundle during the build. Didn't want to muddy this diff with that, though